tests/integration/actions/exec/base.py

```diff
@@ -0,0 +1,110 @@
+"""The base class for exec interactive and stdout tests."""
+
+import difflib
+import json
+import os
+
+from pathlib import Path
+from typing import Generator
+from typing import Union
+
+import pytest
+
+from ..._common import fixture_path_from_request
+from ..._common import update_fixtures
+from ..._tmux_session import TmuxSession
+from ..._interactions import SearchFor
+from ..._interactions import Step
+from ....defaults import FIXTURES_DIR
+
+TEST_FIXTURE_DIR = Path(FIXTURES_DIR, "integration", "actions", "exec")
```

Comment:
WOW, I somehow didn't realize you could pass separate paths to the initializer. TIL